packages/trpc/server/routers/viewer/teams/get.schema.ts

```diff
@@ -2,6 +2,7 @@ import { z } from "zod";
 
 export const ZGetInputSchema = z.object({
   teamId: z.number(),
+  isOrg: z.boolean().default(false),
```

Comment:
Can we infer this from the request context? Or are there places where this isn't needed?